pysollib/gamedb.py

```diff
@@ -479,7 +479,7 @@ def _callback(gi, gt=game_type):
         ('fc-2.8',   (343001,)),
         ('fc-2.12',   tuple(range(774, 811)) + (16681,) +
          tuple(range(22217, 22219))),
-        ('fc-2.14', tuple(range(811, 827)))
+        ('fc-2.14', tuple(range(811, 828)))
```

Comment:
Thanks for doing this.  However, 2.14 was released last week, and this game would be the first for the next release (likely 2.16).  I can update this later if needed.

Reply:
Also, with Churchill having an ID of 828, the top range of the tuple should be 829.

Reply:
I removed this completely, so we can adjust it when it's merged.

Reply:
Fair enough.